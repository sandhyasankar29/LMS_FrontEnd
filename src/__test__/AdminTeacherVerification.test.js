import React from 'react';
import ReactDOM from 'react-dom';
import AdminVerify from '../container/AdminTeacherVerify';
import renderer from 'react-test-renderer'

it ("renders without crashing",()=>{
    const div=document.createElement("div");
    ReactDOM.render(<AdminVerify></AdminVerify>,div)
})
it("matches snapshot", () => {
    const tree = renderer.create(<AdminVerify></AdminVerify>).toJSON();
    expect(tree).toMatchSnapshot();
})