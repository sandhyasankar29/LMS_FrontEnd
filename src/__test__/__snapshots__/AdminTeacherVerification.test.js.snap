// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="StyledGrommet-sc-19lkkz7-0 jirFMY"
>
  <header
    className="StyledBox-sc-13pk1d4-0 jwhdBG"
  >
    <div
      className="StyledBox-sc-13pk1d4-0 dfYOnT"
    >
      <svg
        aria-label="User"
        className="StyledIcon-ofa7kd-0 dcacLZ"
        viewBox="0 0 24 24"
      >
        <path
          d="M8,24 L8,19 M16,24 L16,19 M3,24 L3,19 C3,14.0294373 7.02943725,11 12,11 C16.9705627,11 21,14.0294373 21,19 L21,24 M12,11 C14.7614237,11 17,8.76142375 17,6 C17,3.23857625 14.7614237,1 12,1 C9.23857625,1 7,3.23857625 7,6 C7,8.76142375 9.23857625,11 12,11 Z"
          fill="none"
          stroke="#000"
          strokeWidth="2"
        />
      </svg>
      <div
        className="StyledBox__StyledBoxGap-sc-13pk1d4-1 eBVJAG"
      />
      <a
        className="StyledAnchor-sc-1rp7lwl-0 ePWGdF"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        Welcome Admin
      </a>
    </div>
    <div
      className="StyledBox__StyledBoxGap-sc-13pk1d4-1 ffHQai"
    />
    <nav
      className="StyledBox-sc-13pk1d4-0 glIhNO"
    >
      <a
        className="StyledAnchor-sc-1rp7lwl-0 eaoGTv"
        href="/adminteacherverify"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        Verify Teacher
      </a>
      <div
        className="StyledBox__StyledBoxGap-sc-13pk1d4-1 ffHQai"
      />
      <a
        className="StyledAnchor-sc-1rp7lwl-0 eaoGTv"
        href="/adminrevoke"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        Revoke Access
      </a>
      <div
        className="StyledBox__StyledBoxGap-sc-13pk1d4-1 ffHQai"
      />
      <button
        className="StyledButton-sc-323bzc-0 bZMlsb"
        id="logoutBtn"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        type="button"
      >
        <svg
          aria-label="Logout"
          className="StyledIcon-ofa7kd-0 eBBIEF"
          viewBox="0 0 24 24"
        >
          <path
            d="M13,9 L13,2 L1,2 L1,22 L13,22 L13,15 M22,12 L5,12 M17,7 L22,12 L17,17"
            fill="none"
            stroke="#231F20"
            strokeWidth="2"
          />
        </svg>
      </button>
    </nav>
  </header>
  <div
    className="StyledBox-sc-13pk1d4-0 jsieLm"
  >
    <div
      className="f2 w-100 pa2"
    >
      <span
        className="StyledText-sc-1sadyjn-0 DkZvS"
      >
        Fetch the Teachers Details for Registration Approval
      </span>
    </div>
    <div
      className="f2 w-100 pa2"
    >
      <button
        className="StyledButton-sc-323bzc-0 jfCvHs"
        data-testid="button"
        id="fetchBtn"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        type="button"
      >
        <div
          className="StyledBox-sc-13pk1d4-0 iQEJEP"
        >
          <svg
            aria-label="Update"
            className="StyledIcon-ofa7kd-0 dcacLZ"
            viewBox="0 0 24 24"
          >
            <path
              d="M1.7507,16.0022 C3.3517,20.0982 7.3367,23.0002 11.9997,23.0002 C18.0747,23.0002 22.9997,18.0752 22.9997,12.0002 M22.2497,7.9982 C20.6487,3.9012 16.6627,1.0002 11.9997,1.0002 C5.9247,1.0002 0.9997,5.9252 0.9997,12.0002 M8.9997,16.0002 L0.9997,16.0002 L0.9997,24.0002 M22.9997,0.0002 L22.9997,8.0002 L14.9997,8.0002"
              fill="none"
              stroke="#000"
              strokeWidth="2"
            />
          </svg>
          <div
            className="StyledBox__StyledBoxGap-sc-13pk1d4-1 hjorfk"
          />
          Fetch
        </div>
      </button>
    </div>
  </div>
  <div
    className="StyledBox-sc-13pk1d4-0 bxevCE"
  >
    <div
      className="StyledBox-sc-13pk1d4-0 bJtzEt"
      style={
        Object {
          "marginTop": "10%",
        }
      }
    >
      <img
        className="StyledImage-ey4zx9-0 iBeQsm"
        src="empty.svg"
      />
    </div>
  </div>
</div>
`;
